fix #1

vivaxy opened the pull request:
use loacation.hash to store page status, therefore posts will have their own links

vivaxy committed:
add page link

vivaxy committed:
optimize

Wferr commented:
With the latest version of github-pages they are dropping support for all markdown engines besides kramdown (see https://github.com/blog/2100-github-pages-now-faster-and-simpler-with-jekyll-3-0 ) So after fixing that I would be glad to merge it in.

vivaxy committed:
use kramdown since github support GFM within kramdown

vivaxy commented:
1. about link
   URL for post is like `{{ site.baseurl | prepend: site.url }}#{{ post.url }}`
   eg. `http://127.0.0.1:4000/#/2015/04/25/workshop`
   actions:
   a. open card -> change hash to `post.id`
   b. close card -> change hash to `''`
   render page according to hash change event
2. about markdown
   Since github kramdown support GFM, I have fixed that.
3. about feed.xml
   Added `feed.xml` in this theme.

Wferr commented:
Awesome, thanks so much. Improved usability of this template by aton.
